ql/src/test/queries/clientpositive/auth_create_ext_table_1.q

```diff
@@ -0,0 +1,15 @@
+dfs ${system:test.dfs.mkdir} ${system:test.tmp.dir}/a_ext_create_tab1;
+dfs -touchz ${system:test.tmp.dir}/a_ext_create_tab1/1.txt;
+dfs -chmod 555 ${system:test.tmp.dir}/a_ext_create_tab1/1.txt;
+
+dfs ${system:test.dfs.mkdir} ${system:test.tmp.dir}/a_ext_create_tab2;
+dfs -chmod 555 ${system:test.tmp.dir}/a_ext_create_tab2;
+
+set hive.metastore.pre.event.listeners=org.apache.hadoop.hive.ql.security.authorization.plugin.metastore.HiveMetaStoreAuthorizer;
+set hive.security.authorization.manager=org.apache.hadoop.hive.ql.security.authorization.plugin.fallback.FallbackHiveAuthorizerFactory;
```

Comment:
I think it would be nice to have a negative qtest with CTAS query just to show that write permissions are honored during insert on a table.

Reply:
@saihemanth-cloudera I agree with the thought of having a negative test for CTAS. Since the exception is thrown at runtime while trying to create a staging directory, the q.out file will not have any error/exception. Will it justify a valid negative test scenario? I was not sure about this and excluded the test.